OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -56,13 +55,8 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Ru
 
 			var chatPanel = (ContainerWidget)widget;
 			chatOverlay = chatPanel.GetOrNull<ContainerWidget>("CHAT_OVERLAY");
-			if (chatOverlay != null)
-			{
-				chatOverlayDisplay = chatOverlay.Get<ChatDisplayWidget>("CHAT_DISPLAY");
-				chatOverlay.Visible = false;
-			}
-			else
-				inDialog = true;
+			chatOverlayDisplay = chatOverlay.Get<ChatDisplayWidget>("CHAT_DISPLAY");
```

Comment:
`chatOverlay` may be null as it is GetOrNull before. This will trigger a Coverity warning. Either keep the null check or change GetOrNull to Get above.